Volha_Ivanova/Module_1_Java/Classes_and_Objects/src/com/epam/cdp/volha_ivanova/task1/sweets/Gift.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.cdp.volha_ivanova.task1.sweets;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+
+import com.epam.cdp.volha_ivanova.task1.sweets.candies.chocolateCandies.ChocolateCandies;
+import com.epam.cdp.volha_ivanova.task1.sweets.candies.lollipop.Lollipop;
+import com.epam.cdp.volha_ivanova.task1.sweets.chocolate.Chocolate;
+import com.epam.cdp.volha_ivanova.task1.sweets.waffles.Waffles;
+
+public class Gift {	
+	
+	int giftWeight;			
+	Scanner scanner = new Scanner(System.in);
+	Waffles waffle = new Waffles("Waffles", 10, scanner.nextInt());	
+	Chocolate chocolate1 = new Chocolate("White chocolate", "white", 100, scanner.nextInt());	
```

Comment:
if you want to have several types of chocolate need to add field type instead of using name. Name it's just name of candies.
But again/ in your current case it name of set of candies (my recommendation in comments below)